src/models/appsmodel.cpp

```diff
@@ -190,6 +190,12 @@ QVariant AppsModel::data(const QModelIndex &index, int role) const
         else if (!firstChar.isLetter()) return QString("&%1").arg(transliterated);
         return transliterated;
     }
+    case AppsModel::AllTransliteratedRole: {
+        // it's useful to search(e.g. Music: YinYue or YinLe -> YinYueYinLe)
+        const auto decodedDisplay = Dtk::Core::pinyin(index.data(Qt::DisplayRole).toString(), Dtk::Core::TS_NoneTone);
+        const QString &transliterated = decodedDisplay.join("");
```

Comment:
```suggestion
        const QString &transliterated = decodedDisplay.join(".");
```

中间要不要加个不常用的字符当分隔符？避免“leyin”（Yin<ins>leYin</ins>yue）也能匹配到“音乐”的情况

Reply:
好的，